builtin/logical/ssh/storage.go

```diff
@@ -0,0 +1,385 @@
+// Copyright (c) 2024 OpenBao a Series of LF Projects, LLC
+// SPDX-License-Identifier: MPL-2.0
+
+package ssh
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	uuid "github.com/hashicorp/go-uuid"
+	"github.com/openbao/openbao/sdk/v2/helper/errutil"
+	"github.com/openbao/openbao/sdk/v2/logical"
+)
+
+const (
+	defaultRef = "default"
+
+	migrationLogKey     = "config/migrationLog"
+	issuerPrefix        = "config/issuer/"
+	storageIssuerConfig = "config/issuers"
+
+	caPublicKey                       = "ca_public_key"
+	caPrivateKey                      = "ca_private_key"
+	caPublicKeyStoragePath            = "config/ca_public_key"
+	caPublicKeyStoragePathDeprecated  = "public_key"
+	caPrivateKeyStoragePath           = "config/ca_private_key"
+	caPrivateKeyStoragePathDeprecated = "config/ca_bundle"
+
+	maxRolesToScanOnIssuerChange = 100
+	maxRolesToFindOnIssuerChange = 10
```

Comment:
Thoughts on these values for SSH? I copied them from PKI but might be too high here?

Reply:
I think it makes sense, shouldn't be too much work to do.